src/test/java/org/jenkinsci/plugins/oic/OicSecurityRealmTest.java

```diff
@@ -65,20 +66,20 @@ public void testAuthenticate_withUsernamePasswordAuthenticationToken() throws Ex
     }
 
     @Test
-    public void testGetAuthenticationGatewayUrl() throws IOException {
+    public void testGetAuthenticationGatewayUrl() throws IOException, Descriptor.FormException {
```

Comment:
FYI: generally for test methods I would just use `throws Exception` to reduce any maintenance burden